indexing-service/src/test/java/org/apache/druid/indexing/common/task/CompactionTaskTest.java

```diff
@@ -2015,6 +2100,37 @@ public SettableColumnValueSelector<?> makeNewSettableColumnValueSelector()
 
   }
 
+  /**
+   * A class to mimic MSQCompactionRunner behaviour, since the actual class resides in the MSQ extn
```

Comment:
How are we mimicking the behaviour? The methods seem to return only null.

Reply:
Just mimics for validations. Updated the comment